tests/backup/backup_upgrade_test.go

```diff
@@ -504,6 +534,19 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 				log.FailOnError(err, "Switching context to source cluster failed")
 			}
 		})
+		Step("Validating the backup type after upgrade for older backups", func() {
```

Comment:
should we have version check for this verification as well.

Reply:
Yes. Will do